NAMESPACE

```diff
@@ -1,7 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
 export(IdTaxaToDataFrame)
-export(TAXONOMY_RANKS)
```

Comment:
TAXONOMY_RANKS is not visible anymore for user. It is still used in the functions to determine taxonomy ranks. Moreover, user can still modify the value of it:

```
> getTaxonomyRanks()
 [1] "domain"       "superkingdom" "kingdom"      "phylum"       "class"        "order"       
 [7] "family"       "genus"        "species"      "strain"      
> setTaxonomyRanks(c("one", "two"))
> getTaxonomyRanks()
[1] "one" "two"
```